src/main/java/frc/robot/RobotContainer.java

```diff
@@ -38,6 +39,9 @@ public enum DriveType {
     TANK
   }
 
+  private final LoggedDashboardNumber shooterSpeedInput =
+      new LoggedDashboardNumber("Shooter Speed", 300.0);
+
```

Comment:
Should this be in the shooter subsystem and/or shooter default command?

Reply:
If it is moved from here it cannot be used to give input when declaring the command.  I think it should stay here?

Reply:
Ah, I see...You're using an inline command to call runVelocity() with the value.  Since this is just for debug. Fine with me.  I'm guessing the final code will have set/fixed RPM values depending on whether we are shooting in the speaker or amp.